internal/ir/ir.go

```diff
@@ -122,8 +122,9 @@ type Signature struct {
 // The Parameter implements Value interface.
 type Parameter struct {
 	node
-	name  string // Name of parameter.
-	Value Value  // Default value of parameter or nil.
+	name     string // Name of parameter.
+	Value    Value  // Default value of parameter or nil.
+	Receiver bool   // True if parameter is the receiver of a class
```

Comment:
I think that this comment is better: "True if parameter is a receiver of a struct method."